web-components/src/components/shared/zoomable-image.ts

```diff
@@ -22,75 +38,98 @@ export class ZoomableImage extends LitElement {
     css`
       :host {
         display: block;
+        width: 100%;
       }
-      .panzoom-parent {
-        position: relative;
+      .zoomable-image {
+        margin: 0 auto;
+      }
+      .cropper-parent {
         position: relative;
+        overflow: hidden;
         border: 1px solid black;
         background-color: white;
       }
-      .panzoom {
-        position: relative;
+
+      .button-container {
+        display: flex;
+        justify-content: end;
+        gap: 0.5rem;
+        margin-top: 0.5rem;
       }
-      .panzoom img {
-        display: block;
-        width: 100%;
-        height: 100%;
-        cursor: zoom-in;
+      .crop-result-title {
+        font-weight: bold;
+        text-align: center;
+        margin-top: 1rem;
+        margin-bottom: 0.5rem;
+      }
+      .crop-result-buttons {
+        margin-top: 0.5rem;
       }
     `,
     FLEX,
-    getButtonClasses([ButtonType.LINK]),
+    getButtonClasses([ButtonType.White, ButtonType.Chocolate, ButtonType.LINK]),
   ]
 
-  /*
-   * Panzoom element
-   */
-  @query(".panzoom")
-  element!: HTMLElement
+  @query("cropper-canvas")
+  canvasElement!: CropperCanvas
 
-  /*
-   * Image element
-   */
-  @query("img")
-  imageElement!: HTMLImageElement
+  @query("cropper-image")
+  imageElement!: CropperImage
+
+  @query("cropper-selection")
+  selectionElement!: CropperSelection
 
   /*
    * Panzoom instance
```

Comment:
I think you still need a search / replace ;-) 
```suggestion
   * cropperjs instance
```